.github/workflows/publish-draft-releases.yml

```diff
@@ -496,17 +402,17 @@ jobs:
             id: calamari-local
             expected:
               block-count:
-                relay: 25
-                para: 6
+                relay: 20
+                para: 2
               peer-count:
                 relay: 5
                 para: 2
           -
             id: calamari-dev
             expected:
               block-count:
-                relay: 25
-                para: 6
+                relay: 20
```

Comment:
Any reason for this change?

Reply:
This test is sometimes failing for some reason.
My hunch was that it didn't have enough time to reach these blocks, so I tried reducing it.
But it seems this might not be the original reason, as it still failed once.

Reply:
Ok I just realized the actual reason. Sometimes blocks are not logged as finalized and their finalization is implied from succeeding blocks being finalized.
And sometimes this is the block number we're look to parse.
Anyway i'm going to add a fix for this in a separate PR.